src/util/localStorage.ts

```diff
@@ -0,0 +1,24 @@
+import _ from 'lodash'
+import { IStorage } from '../types'
+
+let storage: IStorage = {
+  clear(): void {
+    localStorage.clear()
+  },
+
+  getItem(key: string): string | null {
```

Comment:
This `getItem` returns `string | null` while the native version returns `Promise<string | null>`. I think they should both return `Promise<string | null>`. If web and native have a different interface, it defeats the purpose of abstracting away the `localStorage` implementation. Same for the other functions.

Reply:
💯, will refactor it.

Reply:
we just made async store sync to prevent changing to async call the whole app.